examples/torchrec_dlrm/dlrm_handler.py

```diff
@@ -0,0 +1,147 @@
+"""
+Handler for Torchrec DLRM based recommendation system
+"""
+import json
+import logging
+import os
+from abc import ABC
+
+import torch
+from torchrec.datasets.criteo import DEFAULT_CAT_NAMES
+from torchrec.sparse.jagged_tensor import KeyedJaggedTensor
+
+from ts.torch_handler.base_handler import BaseHandler
+
+logger = logging.getLogger(__name__)
+
+
+class TorchRecDLRMHandler(BaseHandler, ABC):
+    """
+    Handler for TorchRec DLRM example
+    """
+
+    def initialize(self, context):
+        """Initialize function loads the model.pt file and initialized the model object.
+           This version creates and initialized the model on cpu fist and transfers to gpu in a second step to prevent GPU OOM.
+        Args:
+            context (context): It is a JSON Object containing information
+            pertaining to the model artifacts parameters.
+        Raises:
+            RuntimeError: Raises the Runtime error when the model.py is missing
+        """
+        properties = context.system_properties
+
+        # Set device to cpu to prevent GPU OOM errors
+        self.device = "cpu"
```

Comment:
would we be able to use the "meta" device here as well?

Reply:
I am not sure about this. We can revisit once @mreso is back?